data/human/culture conversations.txt

```diff
@@ -120,6 +149,38 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Forest Fire"
+	minor
+	source
+		government "Free Worlds"
+		attributes "frontier"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning smoke permeates the air as you step off your craft, causing you to rush towards the nearest shelter, coughing along the way. Once there, you find people shouting, rushing about to get tape plastered onto the windows. No one seems to have noticed your arrival and none give an explanation.`
+			choice
+				`	"Hey! What's happening? I can help!" (Shout.)`
+					goto help
+				`	(Look for someone not in motion.)`
+			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or one some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`
```

Comment:
> You seek out a pregnant lady sitting on a chair, a mask worn as she explains:

The "a mask worn" caught me off guard. is that valid?

Reply:
It may or may not be valid, depending on how you interpreted it.
Worst case scenario, just says "You seek out a masked pregnant lady" or "You seek out a masked lady sitting on a chair, hands placed protectively over her belly" or some other ways to portray the idea.